scripts/deploy.ts

```diff
@@ -69,6 +68,7 @@ task('deploy:amplforce:testnet', 'Deploy ampleforth contract suite for testnet')
       RATE_REPORT_EXPIRATION_SEC,
       RATE_REPORT_DELAY_SEC,
       RATE_MIN_PROVIDERS,
+      1 // TODO - NEEDS ELABORATION
```

Comment:
where is this value coming from? 

1 sounds for me like the base for CPI/inflation. Without having any further context.

Reply:
was just added to make it run, as current constructor is built that way

Reply:
Hence, the comment